### PR TITLE
Add AppsAndFeaturesEntries to GIGABYTE.GigabyteControlCenter version …

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
…24.12.06.01

- Resolve https://github.com/microsoft/winget-pkgs/pull/200457#issuecomment-2560408838

Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/201791)

### CODE REVIEW

File: manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml [hunk 1]
```diff
@@ -19,5 +19,9 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://download.gigabyte.cn/FileList/Utility/GCC_24.12.06.01.zip
```
Comment (rakleed) [inline manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml:20]: Better to use this link: https://download.gigabyte.com/FileList/Utility/GCC_24.12.06.01.zip

File: manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml [hunk 2]
```diff
@@ -19,5 +19,9 @@ Installers:
...
   InstallerSha256: 3A7BC03B6154E221DFD0B0E1E6847C2B65DB549817530D88B0F1C8FFC74224E1
+  AppsAndFeaturesEntries:
+  - DisplayName: GIGABYTE Control Center 24.12.06.01
+    DisplayVersion: 24.12.06.01
```
Comment (rakleed) [inline manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml:24]: It should be `24.10.28.01`:

CleanShot 2024-12-24 at 03 05 45

**Review by rakleed — CHANGES_REQUESTED** [review]

File: manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml (hunk 2 above)
Comment (DuckDuckStudio) [inline manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml:24]:
```suggestion
  AppsAndFeaturesEntries:
  - DisplayName: GIGABYTE Control Center 24.10.28.01
    DisplayVersion: 24.10.28.01
```

File: manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml (hunk 1 above)
Comment (DuckDuckStudio) [inline manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml:20]:
```suggestion
  InstallerUrl: https://download.gigabyte.com/FileList/Utility/GCC_24.12.06.01.zip
```

**Comment by DuckDuckStudio** [comment]
Done.

**Review by rakleed — APPROVED** [review]

**Comment by rakleed** [comment]
> Would this be better?
> 
> ```yaml
> Installers:
> - Architecture: x64
>   InstallerUrl: https://download.gigabyte.com/FileList/Utility/GCC_24.12.06.01.zip
>   InstallerSha256: 3A7BC03B6154E221DFD0B0E1E6847C2B65DB549817530D88B0F1C8FFC74224E1
>   AppsAndFeaturesEntries:
>   - DisplayName: GIGABYTE Control Center 24.10.28.01
>     DisplayVersion: 24.10.28.01
>     Publisher: GIGABYTE
> - Architecture: x64
>   InstallerLocale: zh-CN
>   InstallerUrl: https://download.gigabyte.cn/FileList/Utility/GCC_24.12.06.01.zip
>   InstallerSha256: 3A7BC03B6154E221DFD0B0E1E6847C2B65DB549817530D88B0F1C8FFC74224E1
>   AppsAndFeaturesEntries:
>   - DisplayName: GIGABYTE Control Center 24.10.28.01
>     DisplayVersion: 24.10.28.01
>     Publisher: GIGABYTE
> ```

I didn't use different locales, it's better to wait for @stephengillie feedback.

**Comment by stephengillie** [comment]
When the `DisplayVersion` when it matches the `PackageVersion`, then the `DisplayVersion` should be left out. CC @mdanish and @Trenly for more background.

**Comment by rakleed** [comment]
@DuckDuckStudio I compared the hashes of the two files and they are the same.

`gigabyte.com` is not blocked in mainland China. I also tried downloading through a Chinese VPN, but in any case, the download speed was always several times lower if you downloaded from `.cn.` Do you have the same? If yes, then you can leave it as is.

@stephengillie 

> When the DisplayVersion when it matches the PackageVersion, then the DisplayVersion should be left out. CC @mdanish and @Trenly for more background.

In the quoted message, the incorrect version was specified in `AppsAndFeaturesEntries`, but in the modified file the version was specified correctly, and it differs from `PackageVersion`: https://github.com/microsoft/winget-pkgs/pull/201791/files

The question was whether adding a version for another language to `installer` was necessary. However, it seems that this is not relevant now (see a little higher in this message).

**Comment by DuckDuckStudio** [comment]
> @DuckDuckStudio I compared the hashes of the two files and they are the same.
> 
> `gigabyte.com` is not blocked in mainland China. I also tried downloading through a Chinese VPN, but in any case, the download speed was always several times lower if you downloaded from `.cn.` Do you have the same? If yes, then you can leave it as is.
> 

Thanks for finding this out, it did for me...
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**